src/js/actions/userActions.js

```diff
@@ -62,6 +62,8 @@ export const loginUser = userData => (dispatch, getState) =>
       return Promise.all([Promise.reject(err), dispatch(handleLoginError(err, has2FA))]);
```

Comment:
finally the tfa token entryfield is not visible on logout! (also tested it, works perfectly)